.dart_tool/package_config.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
Is there are a tool or package other than `pub` that knows how to generate a `package_config.json` file with the right schema?

Reply:
The `package_config` package has a `toJson` method on `PackageConfig`. I am not sure what the API for creating PackageConfig objects is like, but you do use that. Ultimately it would just be more complicated I think than maintaining this file.